src/Roslyn.Suppressions/Roslyn.Suppressions/SettingsFile/RoslynSettingsFileStorage.cs

```diff
@@ -19,82 +19,83 @@
  */
 
 using System;
-using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using System.Diagnostics;
 using System.IO.Abstractions;
-using System.Linq;
 using Newtonsoft.Json;
-using SonarLint.VisualStudio.Core.Suppressions;
 using SonarLint.VisualStudio.Integration;
 using SonarLint.VisualStudio.Roslyn.Suppressions.Resources;
-using SonarQube.Client.Models;
 
 namespace SonarLint.VisualStudio.Roslyn.Suppressions.SettingsFile
 {
-    [Export(typeof(ISuppressedIssuesFileStorage))]
-    internal class SuppressedIssuesFileStorage : ISuppressedIssuesFileStorage
+    public interface IRoslynSettingsFileStorage
+    {
+        void Update(RoslynSettings settings);
+
+        /// <summary>
+        /// Return the settings for the specific settings key, or null
+        /// if there are no settings for that key
```

Comment:
The file interface will return null if there are no settings: the cache interface will return `RoslynSettings.Empty`.